sql-select/actor-forty.sql

```diff
@@ -0,0 +1,3 @@
+select *
+  from "actors"
+  where "actorId" = '40';
```

Comment:
Don't quote booleans. This works only because Postgres aggressively converts data types.